core/src/main/scala/org/apache/spark/util/Utils.scala

```diff
@@ -71,6 +72,11 @@ private[spark] object CallSite {
 private[spark] object Utils extends Logging {
   val random = new Random()
 
+  // Use a common file numbering format which defaults to 5 digits for saving various part files.
+  val numFormatter = NumberFormat.getIntegerInstance()
```

Comment:
This is a global utility class; it shouldn't have an object so specific to one usage